### DISCUSSION TITLE
Welcome to ha-renogy Discussions!

### DISCUSSION THREAD

**Discussion by firstof9** [firstof9/ha-renogy#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jiresongit** [comment]
Amazing work! Can not believe how long I've been waiting for this!

**Comment by nath69uk** [comment]
Great work! Really interesting being able to have a quick glance HA to see whats going on in our Motorhome.

Is there a way to split out the solar data on its own? Trying to be able to see what the solar is generating.

EDIT## The load values also seem to not work. I presume this is an issue with Renogy and what data is sent via the API?

**Reply by firstof9** [reply]
Any missing data is going to be missing data from the API for the most part.

**Reply by nath69uk** [reply]
I raised an issue with Renogy today, direct connecting to the dc2dc I can see solar and the data. If I connect the dc2dc to my network and connect that way the solar values are empty and that's with the dc home app.

![WhatsApp Image 2025-02-27 at 12 26 21_10aa16a6](https://github.com/user-attachments/assets/754e54db-7a5c-4947-a519-b4cd97ef10fe)

**Reply by firstof9** [reply]
They have a discord server setup too, @Anto79-ops has the link somewhere

**Reply by Anto79-ops** [reply]
Here you go:

https://discord.gg/WX5Gxdz5

**Reply by WS-Dave** [reply]
looks like the discord invite has expired... :-(

**Reply by Anto79-ops** [reply]
https://discord.gg/dD2xfRbG

Try this one
